src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
+    public static final String FIRST_KEYWORD = "supply";
+    public static final String SECOND_KEYWORD = "buy";
+    public static final String RESULT_KEYWORD = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder outputString = new StringBuilder();
+        try (BufferedReader inputFileBuffRdr = new BufferedReader(new FileReader(fromFileName))) {
+            String inputLine;
+            int[] countedBalances = {0, 0};
+            while ((inputLine = inputFileBuffRdr.readLine()) != null) {
+                String[] splittedInputLine = inputLine.split(",");
+                if (splittedInputLine[0].equals(FIRST_KEYWORD)) {
+                    countedBalances[0] += Integer.parseInt(splittedInputLine[1]);
+                } else {
+                    countedBalances[1] += Integer.parseInt(splittedInputLine[1]);
```

Comment:
The current logic assumes that any line not starting with 'supply' is a 'buy' entry. This might not be accurate if there are other types of entries or malformed lines. Consider adding a check to ensure that the line starts with a recognized keyword before processing.